### PR TITLE
RIA-3876:Out of country screening questions

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-2110

### Change description ###
see ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: e2e/features/stepDefinitions/ccd-page.steps.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ Given('I create a new case', async function () {
     await ccdPage.doesDropdownHaveValues('Event');
     await ccdFormPage.setFieldValue(
         'Case type',
-        'Appeal* master'
+        'Appeal* RIA-2110'
```
Comment (Lee365) [inline e2e/features/stepDefinitions/ccd-page.steps.ts:22]: This need to be reverted.

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: e2e/features/stepDefinitions/ccd-page.steps.ts (hunk 1 above)
Comment (mkayad) [reply e2e/features/stepDefinitions/ccd-page.steps.ts]: changed to master

**Review by Lee365 — APPROVED** [review]
LGTM

File: e2e/features/start-appeal-out-of-country-no-sponsor.feature [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    Then I should see `DRAFT` for the `Appeal reference` field
+    And I should see `Yes` for the `Out of country` field
+    And I should see `GWF1234567` for the `Global Web form` field
+    And I should see `{$TODAY|D MMM YYYY}` for the `Date of entry clearance decision` field
+
+    When I click the `Appeal` tab
+    Then I should see `DRAFT` for the `Appeal reference` field
+    And I should see `Yes` for the `Out of country` field
+    And I should see `A decision to refuse a human rights claim for entry clearance` for the `Out of country decision` field
+
```
Comment (Srihari-Nakkala) [inline e2e/features/start-appeal-out-of-country-no-sponsor.feature:48]: @mkayad This Test won't work with when we switch ON F&P Flag, AC is covering F&P scenarios as well and Not 
sure why we are not submitting appeal as part of this Tests.

File: e2e/features/start-appeal-out-of-country-no-sponsor.feature [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    And I complete the `Legal representative details` page
+    And I complete the `Start appeal check your answers` page
+
+    Then I should see the text `Your appeal details have been saved`
+    And I should see the text `You still need to submit it`
+    And I should see the text `If you're ready to proceed submit your appeal.`
+    And I should see the text `Not ready to submit yet?`
+    And I should see the text `You can return to the case details to make changes.`
+    And I click the `Close and Return to case details` button
+
```
Comment (Srihari-Nakkala) [inline e2e/features/start-appeal-out-of-country-no-sponsor.feature:78]: Same Here @mkayad

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**